src/matches.rs

```diff
@@ -104,7 +104,7 @@ pub(crate) fn rewrite_match(
         String::new()
     } else {
         inner_attrs
-            .rewrite(context, shape)
+            .rewrite(context, shape.block_indent(context.config.tab_spaces()))
```

Comment:
Let's [version-gate](https://github.com/rust-lang/rustfmt/blob/master/Contributing.md#version-gate-formatting-changes) this formatting fix. You can then add [a comment like`// rustfmt-version: Two` to the start of your test cases](https://github.com/rust-lang/rustfmt/blob/master/Contributing.md#create-test-cases). Feel free to add `version: One` tests if you'd like.